compressor/seata-compressor-lz4/src/main/java/io/seata/compressor/lz4/Lz4Util.java

```diff
@@ -0,0 +1,77 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.compressor.lz4;
+
+import net.jpountz.lz4.LZ4BlockInputStream;
+import net.jpountz.lz4.LZ4BlockOutputStream;
+import net.jpountz.lz4.LZ4Compressor;
+import net.jpountz.lz4.LZ4Factory;
+import net.jpountz.lz4.LZ4FastDecompressor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+/**
+ * the Lz4 Util
+ *
+ * @author diguage
+ */
+public class Lz4Util {
+    private static final Logger LOGGER = LoggerFactory.getLogger(Lz4Util.class);
+    private static final int ARRAY_SIZE = 2048;
+
+    private static final LZ4Factory LZ4_FACTORY = LZ4Factory.fastestInstance();
```

Comment:
This one can get rid of static decoration

Reply:
The `LZ4_FACTORY` is used in the `static` methods.

Reply:
Remove the static variable and use it directly in the method

Reply:
Remove the static variable and use it directly in the method